### PR TITLE
Elastic job plugin use wrong span type

### PR DESCRIPTION
**Pull request by zhaoyuguang**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.
Fix elastic job plugin use wrong span type

### CODE REVIEW

File: apm-sniffer/apm-sdk-plugin/elastic-job-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/elasticjob/ElasticJobExecutorInterceptor.java [hunk 1]
```diff
@@ -36,7 +35,7 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
         ShardingContexts shardingContexts = (ShardingContexts) allArguments[0];
         Integer item = (Integer) allArguments[1];
         String operateName = ComponentsDefine.ELASTIC_JOB.getName() + "/" + shardingContexts.getJobName();
-        AbstractSpan span = ContextManager.createEntrySpan(operateName, new ContextCarrier());
+        AbstractSpan span = ContextManager.createLocalSpan(operateName);
```
Comment (wu-sheng) [inline apm-sniffer/apm-sdk-plugin/elastic-job-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/elasticjob/ElasticJobExecutorInterceptor.java:38]: Do you want to analyze? If you want, add `logic span` tag.

Comment (wu-sheng) [reply apm-sniffer/apm-sdk-plugin/elastic-job-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/elasticjob/ElasticJobExecutorInterceptor.java:38]: @tristaZero @terrymanu After this is added, on the SkyWalking UI, you will have the job metrics, besides the traces, including
1. Execution time, including avg, percentile
1. Successful rate, determined through whether the main entrance has an exception
1. What other endpoints(HTTP/gRPC) this task depends on.

This is a thing we called the logic endpoint.

Comment (tristaZero) [reply apm-sniffer/apm-sdk-plugin/elastic-job-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/elasticjob/ElasticJobExecutorInterceptor.java:38]: It sounds a great feature. Express my thinks to @zhaoyuguang and @wu-sheng .

Comment (zhaoyuguang) [reply apm-sniffer/apm-sdk-plugin/elastic-job-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/elasticjob/ElasticJobExecutorInterceptor.java:39]: Elastic-job`s had specification logic-span

**Review by wu-sheng — APPROVED** [review]

**Review by wu-sheng — APPROVED** [review]

**Comment by wu-sheng** [comment]
image

**Pull request merged**